lib/macho/load_commands.rb

```diff
@@ -1008,4 +1008,68 @@ def version_string
       segs.join(".")
     end
   end
+
+  # An obsolete load command containing the offset and size of the (GNU style)
+  # symbol table information. Corresponds to LC_SYMSEG.
+  class SymsegCommand < LoadCommand
+    # @return [Fixnum] the offset to the symbol segment
+    attr_reader :offset
+
+    # @return [Fixnum] the size of the symbol segment in bytes
+    attr_reader :size
+
+    FORMAT = "L=4"
+    SIZEOF = 16
+
+    # @api private
+    def initialize(view, cmd, cmdsize, offset, size)
+      super(view, cmd, cmdsize)
+      @offset = offset
+      @size = size
+    end
+  end
+
+  # An obsolete load command containing a free format string table. Each string
+  # is null-terminated and the command is null-aligned to a multiple of 4.
```

Comment:
I think “zero-padded” is clearer than “null-aligned” or am I misunderstanding this? Any reason why you omitted the “Corresponds to `LC_*`.” comment here and in the next two commands?